test/test_helper.rb

```diff
@@ -1,6 +1,18 @@
+if ENV['CI'].to_s == 'true'
+  require "codeclimate-test-reporter"
+  CodeClimate::TestReporter.start
```

Comment:
It looks like this implementation is deprecated:
https://github.com/codeclimate/ruby-test-reporter/blob/master/README.md

This could be done with SimpleCov:
https://docs.codeclimate.com/v1.0/docs/configuring-test-coverage

This change could be added to the `.travis.yml`